docs/source/python/development.rst

```diff
@@ -414,3 +416,20 @@ After these steps are completed, the documentation is rendered in HTML
 format in ``docs/_build/html``.  In particular, you can point your browser
 at ``docs/_build/html/index.html`` to read the docs and review any changes
 you made.
+
+
+Building with Docker
+--------------------
+
+You can use Docker to build the documentation:
+
+   .. code-block:: shell
+
+        docker-compose build cpp 
+        docker-compose build python 
+        docker-compose run docs 
```

Comment:
There are too much indentations.

Reply:
Fixed in the new pull request https://github.com/apache/arrow/pull/3281 thanks